(2) Vox Populi/Balance Changes/Text/en_US/CivText.sql

```diff
@@ -552,7 +552,7 @@ WHERE Tag = 'TXT_KEY_UNIT_GREEK_COMPANIONCAVALRY_STRATEGY';
 -- Huns
 --------------------
 UPDATE Language_en_US
-SET Text = 'Friendly and Enemy [ICON_WAR] War Weariness is modified by 50%. Mounted Melee and Armor Units may capture defeated Units, which do not consume Unit Supply. Claim adjacent unowned land tiles when Cities claim a tile of the same type.'
+SET Text = 'Friendly and Enemy [ICON_WAR] [COLOR_POSITIVE_TEXT]War Weariness[ENDCOLOR] is modified by 50%. Mounted Melee and Armor Units may capture defeated Units, which do not consume Unit Supply. Claim adjacent unowned land tiles when Cities claim a tile of the same type.'
```

Comment:
Hun's UA War Weariness Modifier changed to +100% (on Enemy), but still -50% (for Self).

> Hun's UA now has Subsequent Attack Bonus, but no longer has Coercion.
Not listed in UA text.
> Hun's UA War Weariness Modifier changed to +100% (on Enemy), but still -50% (for Self).
Not changed in UA text.



No mention that Horse Archers double up on the UA bonus in any of their text.